android/assets/jsons/Tutorials.json

```diff
@@ -261,13 +261,15 @@
     "⑱: Tile information for the selected hex - current or potential yield, terrain, effects, present units, city located there and such. Where appropriate, clicking a line opens the corresponding civilopedia entry.",
     "⑲: Notifications - what happened during the last 'next turn' phase. Some are clickable to show a relevant place on the map, some even show several when you click repeatedly.",
     "⑳: The Next Turn Button - unless there are things to do, in which case the label changes to 'next unit', 'pick policy' and so on.",
+    "㉑: The Multiplayer Button - Here will help you easily check your current multiplayer game.",
     "ⓐ: The overview button leads to the empire overview screen with various tabs (the last one viewed is remembered) holding vital information about the state of your civilization in the world.",
     "ⓑ: The ♪Culture icon shows accumulated ♪Culture and ♪Culture needed for the next policy - in this case, the exclamation mark tells us a next policy can be enacted. Clicking is another way to the policies manager.",
     "ⓒ: Your known strategic resources are displayed here with the available (usage already deducted) number - click to go to the resources overview screen.",
     "ⓓ: Happiness/unhappiness balance and either golden age with turns left or accumulated happiness with amount needed for a golden age is shown next to the smiley. Clicking also leads to the resources overview screen as luxury resources are a way to improve happiness.",
     "ⓔ: The ⍾Science icon shows the number of ⍾Science points produced per turn. Clicking leads to the technology tree.",
     "ⓕ: Number of turns played with translation into calendar years. Click to see the victory overview.",
     "ⓖ: The number of gold coins in your treasury and income. Clicks lead to the Stats overview screen.",
+    "ⓗ: Here you will see what you currently have ☮Faith quantity, of course, if you don't open it ☮Faith Before starts the game, this will be ‘Off’.",
```

Comment:
I'd word this as "The quantity of ☮Faith your citizens have generated, or 'off' if religion is disabled. Clicking it makes you go to the religion overview screen.", which keeps it more in line with the tutorials above in my opinion

Reply:
thank! I have modified

Reply:
I think you're right，thanks you